WalletWasabi.Fluent/Models/Wallets/WalletModel.cs

```diff
@@ -19,6 +19,7 @@ namespace WalletWasabi.Fluent.Models.Wallets;
 public partial class WalletModel : ReactiveObject, IWalletModel
 {
 	private readonly TransactionHistoryBuilder _historyBuilder;
+	private readonly Lazy<IWalletCoinjoinModel> _coinjoin;
```

Comment:
Why is the Lazy?

Reply:
Because `WalletCoinjoinModel` uses `_coinJoinManager = Services.HostedServices.Get<CoinJoinManager>();`, which is null if initialized early.

This might also be related to #11301